.circleci/config.yml

```diff
@@ -5,10 +5,10 @@ orbs:
 #  Xcode version announcments can be found here: https://discuss.circleci.com/c/announcements/
 #  Each post contains a full image manifest, including iOS runtimes, devices, CocoaPods version, etc.
 anchors:
-  - &latest-xcode    "15.3.0"
-  - &latest-ios      "17.4"
-  - &min-ios         "16.4"
-  - &requres-string  "test << matrix.lib >> iOS 16.4"  # Keep this min version up to date
+  - &latest-xcode    "16.1.0"
+  - &latest-ios      "18.1"
+  - &min-ios         "17.5"
```

Comment:
For 12.2, we will test iOS 16.4 manually then?

Reply:
Yeah, I was discussing this with Brianna and it seemed easiest (since I still can't even run anything myself 🙄).  

Github Actions doesn't have this limitation...